pkgs/top-level/all-packages.nix

```diff
@@ -13723,7 +13709,8 @@ let
     retroarch = retroarchBare;
   });
 
-  rssglx = callPackage ../misc/screensavers/rss-glx { };
+  rss-glx = callPackage ../misc/screensavers/rss-glx { };
+  rssglx = builtins.trace "WARNING: rssglx package was renamed to rss-glx" pkgs.rss-glx;
```

Comment:
Don't do this kind of rename, it's meaningless. There are so many packages in the same situation that it's not worth it.